src/index.tsx

```diff
@@ -23,7 +23,7 @@ function Accordion({
   prefix = 'vtex-components',
 }: PropsWithChildren<Props>) {
   const [activeKeys, setActiveKeys] = useState<string[]>([])
-  const prefixClassName = `${prefix}-accordion`
+  const variant = `${prefix}.accordion`
```

Comment:
it would be nice if you called this const `customVariant` and the `prefix` prop was called `variant`. This is how we are using it as a standard within the store.